package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "compile": "npx hardhat compile",
-    "foundry-test": "forge test --fork-url $(grep ARCHIVE_NODE_URL_L2 .env | cut -d '=' -f2) --fork-block-number 9000000 --etherscan-api-key $(grep ETHERSCAN_API_KEY .env | cut -d '=' -f2)",
+    "foundry-test": "forge test --fork-url $(grep ARCHIVE_NODE_URL_L2 .env | cut -d '=' -f2) --fork-block-number 9000000 --etherscan-api-key $(grep ETHERSCAN_API_KEY .env | cut -d '=' -f2) --gas-report",
```

Comment:
Unit tests should really run in a non-forked environment (testing pure logic instead of external integrations). (`foundry-test:unit`)

We can definitely add fork tests with foundry when needed, but we should have a clear distinction between tests that run on a fork and tests that don't.
(`foundry-test:fork`) or (`foundry-test:integration`)